panther_lights/src/driver_node.cpp

```diff
@@ -134,22 +138,7 @@ void DriverNode::FrameCB(
 
 void DriverNode::SetPowerPin(const bool value) const
```

Comment:
Does this method remain const?

Reply:
The code compiled but I guess it make more sense to make int non const in this case